src/java.desktop/share/classes/java/beans/beancontext/BeanContextChild.java

```diff
@@ -65,6 +65,8 @@
  * @see java.beans.VetoableChangeListener
  */
 
+@SuppressWarnings("removal")
+@Deprecated(forRemoval=true, since="23")
```

Comment:
```suggestion
@Deprecated(since="23", forRemoval=true)
```
The most common order of attributes in JDK is `since` followed by `forRemoval`. Should this package be consistent with other deprecated classes?

https://github.com/openjdk/jdk/blob/5fb5e6c8f04e325cbb782431d51251edde4c2618/src/java.base/share/classes/java/lang/Double.java#L1017-L1019

https://github.com/openjdk/jdk/blob/71c5bbcec7052a8394dd49c0a8c46801adbfcae4/src/java.base/share/classes/java/lang/SecurityManager.java#L316-L317